realm/realm-library/src/androidTest/java/io/realm/RealmJsonTests.java

```diff
@@ -2239,6 +2343,7 @@ private void testRequiredPrimitiveListWithNullValue(String fieldName) throws JSO
             realm.createObjectFromJson(PrimitiveListTypes.class, jsonObject);
             fail();
         } catch (IllegalArgumentException ignored) {
+            ignored.printStackTrace();
```

Comment:
Why this?

Reply:
That was me trying to understand why we were hitting that point. Forgot to remove the printstacks.